version: "3.8"
services:
    sasha_sentence_logger:
        image: shedz/sashav2
        volumes: 
            - ./sentence_logger:/sasha_sentence_logger/sasha_sentence_logger
            - /home/pi/.asoundrc:/etc/asound.conf
        entrypoint: ["python","-m" ,"sasha_sentence_logger"]
        #command: python -m dev
        devices: 
            - "/dev/snd:/dev/snd"
        ports:
            - "4545:4545"
    hass:
        image: shedz/hass:v0.1 #homeassistant/raspberrypi3-homeassistant:stable
        init: true
        network_mode: host
        volumes:
            - ./hass:/config
        ports: 
            - "8123:8123"
        environment: 
            - "TZ=Europe/Berlin"
    rhasspy:
        image: shedz/rhasspy:v0.5
        volumes:
            - /home/pi/.config/rhasspy/profiles:/profiles
        devices:
            - "/dev/snd:/dev/snd"
        entrypoint: ["bash","/usr/lib/rhasspy/bin/rhasspy-voltron", "--user-profiles", "/profiles", "--profile", "en"]
        ports:
            - "12101:12101" 
